install.sh

```diff
@@ -12,6 +12,10 @@ amd_lshw() {
   lshw -c display -numeric -disable network | grep -q 'vendor: .* \[1002\]'
 }
 
+mthreads_lshw() {
```

Comment:
Thanks for adding this, although we don't use, nice to give us this info, since we don't own hardware :)